references/classification/transforms.py

```diff
@@ -21,8 +21,11 @@ class RandomMixup(torch.nn.Module):
 
     def __init__(self, num_classes: int, p: float = 0.5, alpha: float = 1.0, inplace: bool = False) -> None:
         super().__init__()
-        assert num_classes > 0, "Please provide a valid positive value for the num_classes."
-        assert alpha > 0, "Alpha param can't be zero."
+        if not num_classes > 0:
+            raise ValueError("Please provide a valid positive value for the num_classes.")
```

Comment:
Nit In general it's helpful for the error message to indicate which value they got, e.g. below. This isn't the original goal of the PR though, so maybe we can leave that as future improvement.

```suggestion
            raise ValueError(f"Please provide a valid positive value for the num_classes. Got num_classes={num_classes}")
```